marketdata/stream/log.go

```diff
@@ -2,40 +2,44 @@ package stream
 
 import (
 	"log"
-	"os"
 )
 
+// Logger wraps methods for leveled, formatted logging.
 type Logger interface {
 	Infof(format string, v ...interface{})
 	Warnf(format string, v ...interface{})
 	Errorf(format string, v ...interface{})
 }
 
-type stdLog struct {
-	logger *log.Logger
-}
-
-var _ Logger = (*stdLog)(nil)
+type defaultLogger struct{}
 
-func (s *stdLog) Infof(format string, v ...interface{}) {
-	// NOTE: there is no concept of levels in the stdlib log package
-	// For less noise, this default implementation will not print any non-error messages
-	// To see these write your own implementation or use a proper logger,
-	//  e.g. https://github.com/uber-go/zap
+func (*defaultLogger) Infof(format string, v ...interface{}) {
+	log.Printf("INFO "+format, v...)
```

Comment:
Would adding a colon after the level be a bad idea?

Reply:
I don't like it, it looks better without it in my opinion.